chromadb/types.py

```diff
@@ -80,6 +82,7 @@ def __init__(
         tenant: str,
         database: str,
         version: int = 0,
+        log_position: int = 0,
```

Comment:
is this 0-indexed or 1-indexed in log service? Want to avoid the default case being a valid offset